django/views/csrf.py

```diff
@@ -131,4 +130,8 @@ def csrf_failure(request, reason=""):
         'docs_version': get_docs_version(),
         'more': _("More information is available with DEBUG=True."),
     })
+    try:
+        t = loader.get_template(template_name)
+    except TemplateDoesNotExist:
+        t = Engine().from_string(CSRF_FAILURE_TEMPLATE)
```

Comment:
I agree. (However, this should also be changed in the 404/403/500 error views then, shouldn't it?)